dipy/workflows/reconst.py

```diff
@@ -454,18 +479,20 @@ def run(self, input_files, bvalues, bvectors, mask_files,
 
             bvals, bvecs = read_bvals_bvecs(bval, bvec)
             gtab = gradient_table(bvals, bvecs,
-                                  b0_threshold=b0_threshold)
+                                  b0_threshold=b0_threshold, atol=bvecs_tol)
```

Comment:
It occurs to me: would it make more sense to use `atol` instead of `bvecs_tol` for consistency with the gradients module?

Reply:
atol is not really informative especially from the command line.